R/features.R

```diff
@@ -107,6 +107,7 @@ available_tags <- function (feature) {
 
             taglists <- mapply (unique, taglists)
             ret <- taglists [[feature]] %>% sort ()
+            ret <- gsub ("&.+", "", ret)
```

Comment:
```suggestion
            ret <- gsub ("&.*$", "", ret)
```
`*` is fine, as there'll always be something after `&` anyway, but most importantly `$` ensures everything is removed, including terminal characters.